docs(split-view): accessibility analysis docs (#6571)
adobe/spectrum-web-components: CONTRIBUTOR-DOCS/03_project-planning/03_components/split-view/accessibility-migration-analysis.md

- A **numeric input** the user sets to a precise value (volume, zoom, brightness) → a slider component with `role="slider"`. Split view's divider reports a position as a percentage, but it is a **layout** control, not a data-entry widget.
- **Collapsing/expanding a whole panel** as a primary interaction (not just a side effect of dragging to an edge) → a dedicated disclosure or panel-toggle pattern with a labeled button, so the action has a discoverable name distinct from "resize."

### What it is not

- **Not** an [APG window splitter](https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/) *page* implementation — the APG does not currently publish a maintained pattern page for this widget (see [Pattern in the APG](#pattern-in-the-apg)). Split view follows the underlying **separator (interactive/range) role** semantics directly from the WAI-ARIA specification instead of a named APG recipe.
- **Not** a tab or accordion: both panes stay mounted and visible at all times. Resizing never hides a pane's content from the accessibility tree the way a collapsed accordion panel or unselected tab panel would.

---

## ARIA and WCAG context

### Pattern in the APG

> - There is **no** current APG pattern page for a resizable "window splitter." Ozzie the ARIA Authoring Practices Guide previously carried a "Window Splitter" pattern in early drafts, but it was not carried forward as a maintained pattern, and the gap is still an open topic in the working group: see [w3c/aria#1443](https://github.com/w3c/aria/issues/1443), which discusses that neither the plain `slider` role nor the (not-yet-shipped) `splitter` role proposal fully covers a two-dimensional resize handle, and that no dedicated role has been adopted.

Reviewer: > Ozzie the ARIA Authoring Practices Guide...

Do we need to clean this bit up? ✨

Author: Cleaned up

> - There is **no** current APG pattern page for a resizable "window splitter." Ozzie the ARIA Authoring Practices Guide previously carried a "Window Splitter" pattern in early drafts, but it was not carried forward as a maintained pattern, and the gap is still an open topic in the working group: see [w3c/aria#1443](https://github.com/w3c/aria/issues/1443), which discusses that neither the plain `slider` role nor the (not-yet-shipped) `splitter` role proposal fully covers a two-dimensional resize handle, and that no dedicated role has been adopted.

Author: ```suggestion
- There is **no** current APG pattern page for a resizable "window splitter." The ARIA Authoring Practices Guide previously carried a "Window Splitter" pattern in early drafts, but it was not carried forward as a maintained pattern, and the gap is still an open topic in the working group: see [w3c/aria#1443](https://github.com/w3c/aria/issues/1443), which discusses that neither the plain `slider` role nor the (not-yet-shipped) `splitter` role proposal fully covers a two-dimensional resize handle, and that no dedicated role has been adopted.
```

- [ ] Non-text contrast for the divider/gripper passes in default, hover, focus-visible, and `forced-colors` states.
- [ ] Touch/pointer target size for the drag handle is confirmed or documented as an accepted exception.
- [ ] `aXe` (WCAG 2.x tags) runs on default, resizable, collapsible, vertical, and nested stories.
- [ ] ARIA snapshot tests cover non-resizable, resizable, collapsible, vertical, and custom-`label` stories.
- [ ] Manual screen reader pass specifically re-tests the mobile/touch scenario from [SWC-276](https://jira.corp.adobe.com/browse/SWC-276).

---

## References

- [WAI-ARIA 1.2: separator](https://www.w3.org/TR/wai-aria-1.2/#separator)
- [WCAG 2.2](https://www.w3.org/TR/WCAG22/)
- [Using ARIA (read this first)](https://www.w3.org/WAI/ARIA/apg/practices/read-me-first/)
- [APG: slider pattern](https://www.w3.org/WAI/ARIA/apg/patterns/slider/) (large-step key convention referenced above)
- [w3c/aria#1443 — resize handle ARIA semantics discussion](https://github.com/w3c/aria/issues/1443)
- [Nord Health: Resizable handle component](https://nordhealth.design/components/resizable-handle/)

> - External context supplied alongside this analysis but not independently fetchable at the time of writing (blocked by the source site): [Stack Overflow — proper accessibility/ARIA role for a resize handle](https://stackoverflow.com/questions/42981485/proper-accessibility-or-aria-role-for-a-resize-handle), [Lenovo glossary — sizing handle](https://www.lenovo.com/us/en/glossary/sizing-handle/). Re-check these if they become reachable, before relying on any specific claim from them.

Reviewer: ```suggestion
```

> - External context supplied alongside this analysis but not independently fetchable at the time of writing (blocked by the source site): [Stack Overflow — proper accessibility/ARIA role for a resize handle](https://stackoverflow.com/questions/42981485/proper-accessibility-or-aria-role-for-a-resize-handle), [Lenovo glossary — sizing handle](https://www.lenovo.com/us/en/glossary/sizing-handle/). Re-check these if they become reachable, before relying on any specific claim from them.

Author: ````suggestion
- [Nord Health: Resizable handle component](https://nordhealth.design/components/resizable-handle/)
- [Stack Overflow — proper accessibility/ARIA role for a resize handle](https://stackoverflow.com/questions/42981485/proper-accessibility-or-aria-role-for-a-resize-handle)
- [Lenovo glossary — sizing handle](https://www.lenovo.com/us/en/glossary/sizing-handle/)
````
